assets/src/application/ui/display/viewport/with-viewport-match.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { mapValues } from 'lodash';
```

Comment:
Why not kill `lodash`?

Reply:
agree,
I have just copied over this package and wanted first to know if you both are ok with pulling this

Reply:
ya let's not use lodash

Reply:
replaced with `ramda` alternative